common/brave_features.cc

```diff
@@ -19,4 +19,7 @@ const base::Feature kBraveRewards{"BraveRewards",
 #endif
 #endif  // defined(OS_ANDROID)
 
+const base::Feature kGlobalPrivacyControl{"GlobalPrivacyControl",
+                                  base::FEATURE_ENABLED_BY_DEFAULT};
```

Comment:
we want this enabled by default in nightly, right? You can check the release channel in `BraveMainDelegate::BasicStartupComplete` and conditionally add it to the list of enabled features